pkg/validate/raw/validate/source_spec_validator.go

```diff
@@ -432,3 +424,12 @@ func MissingConfigMapKey(o client.Object, name, key string) status.Error {
 		Sprintf("%s field spec.helm.valuesFileRefs.valuesFile must specify an existing data key in the referenced ConfigMap; data key %q not found in ConfigMap %q", kind, key, name).
 		BuildWithResources(o)
 }
+
+// ConfigMapMustBeImmutable reports that a referenced ConfigMap from RSync spec.helm.valuesFileSources is
+// not immutable.
+func ConfigMapMustBeImmutable(o client.Object, name string) status.Error {
```

Comment:
HelmValuesConfigMapMustBeImmutable

Reply:
prefix the other error funcs too. they're helm specific here

Reply:
done